megaparsack-doc/scribblings/megaparsack/reference.scrbl

```diff
@@ -242,6 +242,15 @@ Returns a parser that parses a single character that is in @racket[alphabet].
 
 @history[#:added "1.2"]}
 
+@defthing[any-char/p (parser/c char? char?)]{
+Returns a parser that parses a single character.
+                                                            
+@(parser-examples
+  (eval:check (parse-result! (parse-string (any-char/p) "µ")) #\µ)
```

Comment:
Oh, and as for the build error, at first glance it seems to be that `any-char/p` shouldn’t be surrounded in parentheses here.

Reply:
indeed! but it still does not work, it seems that `any-char/p` is not being provided:

```racket
; after reinstalling megaparsack
> (require megaparsack megaparsack/text)
> (parse-string any-char/p "oi")
; any-char/p: undefined;
;  cannot reference an identifier before its definition
;   in module: top-level
;   internal name: any-char/p
```

even though there's
```racket
(provide (contract-out
....
          [any-char/p (parser/c char? char?)]
....))
```